src/openssh/Makefile

```diff
@@ -0,0 +1,26 @@
+.ONESHELL:
+SHELL = /bin/bash
+.SHELLFLAGS += -e
+
+MAIN_TARGET = openssh-server_$(OPENSSH_VERSION)_$(CONFIGURED_ARCH).deb
+DERIVED_TARGETS = openssh-server-dbgsym_$(OPENSSH_VERSION)_$(CONFIGURED_ARCH).deb
+
+$(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
+	# Obtain openssh: https://salsa.debian.org/ssh-team/openssh/-/tree/debian/1%257.9p1-10+deb10u2
+	rm -rf ./openssh-server
+	git clone https://salsa.debian.org/ssh-team/openssh.git openssh-server
+	pushd ./openssh-server
+	git checkout -b openssh-src -f 6d9ca74c48d9911342c6ca5aaac8a25974fa2619
```

Comment:
does openssh have a tag for this commit id?

Reply:
Yes, they have. The commit id is targeting to tag: `debian/1%257.9p1-10+deb10u2`

Reply:
can you use the tag id or add a comment.

Reply:
Yes, I will update

Reply:
@Blueve I didn't see the the change to checkout tag yet. did you forget to push?

Reply:
I didn't have time yesterday. I will update this PR today.